monai/apps/utils.py

```diff
@@ -241,7 +241,7 @@ def extractall(
         cache_dir = Path(output_dir, _basename(filepath).split(".")[0])
     else:
         cache_dir = Path(output_dir)
-    if cache_dir.exists() and len(list(cache_dir.iterdir())) > 0:
+    if cache_dir.exists() and next(cache_dir.iterdir(), None) is not None:
```

Comment:
Maybe we can also use `monai.utils.first()` here?

Thanks.

Reply:
May have a circular import here